index.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/style.css">
-    <title>Hello world</title>
+    <title>My Portfolio</title>
 </head>
```

Comment:
- [ ] Kindly, use better page titles:
      For instance: Your name portfolio (Rotimiazeez's Portfolio)

Reply:
👍🏽